src/main/java/study/calculator/Delimiter.java

```diff
@@ -0,0 +1,26 @@
+package study.calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimiter {
+
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final Pattern DELLIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private final String delimiter;
+    public Delimiter(String target) {
```

Comment:
가독성을 위해 클래스 변수, 멤버 변수, 메서드 사이에 공백을 추가하면 어떨까요?
```suggestion
    public static final String DEFAULT_DELIMITER = ",|:";
    public static final Pattern DELLIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
    
    private final String delimiter;
    
    public Delimiter(String target) {
```

Reply:
인텔리제이에 `2usage` 이런 것 때문에 안보이고 헷갈릴때가 있더라구요 🥲
<img width="793" alt="image" src="https://github.com/next-step/java-racingcar/assets/52321403/2eb216f6-4cb4-45b1-b60c-08d161683a17">
이러한 경우인데 좋은 방법이 있을까요? 🤔